examples/hybrid/driver.jl

```diff
@@ -32,6 +32,7 @@ zd_rayleigh = parsed_args["zd_rayleigh"]
 α_rayleigh_w = parsed_args["alpha_rayleigh_w"]
 zd_viscous = parsed_args["zd_viscous"]
 κ₂_sponge = parsed_args["kappa_2_sponge"]
+config = parsed_args["config"]
```

Comment:
Let's add config to the simulation instead.

Reply:
Yea, I have a lot of if statements based on config, defining a type seems better.